redisgears_v8_plugin/src/v8_backend.rs

```diff
@@ -207,6 +276,26 @@ impl BackendCtxInterfaceInitialised for V8Backend {
                 let ctx = isolate_scope.new_context(None);
                 let ctx_scope = ctx.enter(&isolate_scope);
 
+                let globals = ctx_scope.get_globals();
+                let propeties = globals.get_own_property_names(&ctx_scope);
+                propeties.iter(&ctx_scope).try_for_each(|v| {
+                    let s = v.to_utf8().ok_or(GearsApiError::new("Failed converting global property name to string"))?;
+                    if !GLOBALS_ALLOW_LIST.contains(s.as_str())
+                    {
```

Comment:
Strange, I configured my vscode to run automatic on save.

Reply:
Seems like cargo fmt pass like this, anyway I moved it up.

Reply:
Ohh I guess its because it is not yet compiled because we need to merge this: https://github.com/RedisGears/v8-rs/pull/22
After that the `cargo fmt` will work